timer/derbynet-timer/src/js/remote_start.js

```diff
@@ -16,7 +16,7 @@ class RemoteStart {
 
   // If possible, trigger a gate release
   remote_start() {
-    if (this.profile_remote_start.has_remote_start()) {
```

Comment:
Actually, I'd just remove the function-call parens: this.profile_remote_start.has_remote_start is the field we want, it's just a boolean and not a function.

Reply:
That was my first fix and I have already verified it resolves the issue. 
I also almost deleted the function but I was not sure if it was called elsewhere so I decided to leave it and use it.